src/Take.Blip.Client/Constants.cs

```diff
@@ -17,5 +17,15 @@ public static class Constants
         public const string POSTMASTER = "postmaster";
 
         public const string DEFAULT_STATE = "default";
+
+        /// <summary>
+        /// Default header containing the user's Identity
+        /// </summary>
+        public const string BLIP_USER_HEADER = "X-Blip-User";
```

Comment:
Sorry by missing this PR... 
Move these constants to Builder project, so these values could be replaced on `ProcessHttpAction` without adding a dependence on Client project

Reply:
Created a new `Constants` class within `Take.Blip.Builder` and moved there, and already replaced on `ProcessHttpAction`